smartapps/cuchen-wifi-cooker/cuchen-wifi-cooker.src/cuchen-wifi-cooker.groovy

```diff
@@ -0,0 +1,242 @@
+/**
+ *  Cuchen Wifi Cooker
+ *
+ *  Copyright 2015 Cuchen
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+ 
+import groovy.json.*
+import static java.util.Calendar.*
+
+definition(
+    name: "Cuchen Wifi Cooker",
+    namespace: "cuchen.wifi.cooker",
+    author: "Cuchen",
+    description: "control your Cuchen Wifi Cooker.",
+    category: "My Apps",
+    iconUrl: "http://lihom.jc-square.com/cuchen.png",
+    iconX2Url: "http://lihom.jc-square.com/cuchen@2x.png"
+    //iconX3Url: "https://s3.amazonaws.com/smartapp-icons/Convenience/Cat-Convenience@2x.png"
+    )
+
+
+preferences {
+	def msg = """Tap 'Next' after you have entered in your Cooker credentials.
+
+Once your credentials are accepted, SmartThings will scan your installation for Cooker."""
+
+	page(name: "selectDevices", title: "Connect Your Cuchen Wifi Cooker to SmartThings", install: true, uninstall: true) {
```

Comment:
This will never show "Next" as no page has been defined as the next page. It will simply install the SmartApp.